### ISSUE TITLE
BUG: indexData is undefined when clicking on nested element

### ISSUE THREAD

**Issue by ianmcburnie** [makeup-js/makeup-navigation-emitter#2]
Repro steps below.

Example 1 source is:

```html
<h2>Example 1</h2>
<div class="widget">
    <ul>
        <li tabindex="0">Item 0</li>
        <li>Item 1</li>
        <li>Item 2</li>
    </ul>
    <label>Index</label>
    <input type="text" class="console" readonly disabled />
</div>
```

Clicking a list item will trigger an event with indexData present.

Change this source to:

```html
<h2>Example 1</h2>
<div class="widget">
    <ul>
        <li tabindex="0"><span>Item 0</span></li>
        <li><span>Item 1</span></li>
        <li><span>Item 2</span></li>
    </ul>
    <label>Index</label>
    <input type="text" class="console" readonly disabled />
</div>
```

Clicking a list item will trigger an event without indexData present (because the event now occurs on the span, not the li).

**Comment by ianmcburnie** [comment]
Workaround is to specify the nested element (the span element in the example above) as part of the item selector option:

i.e. change:

```js
NavigationEmitter.createLinear(el, 'li', options[index])`
```

to

```js
NavigationEmitter.createLinear(el, 'li > span', options[index])
```

Actually, maybe this is how things should be. Manually specified in the selector like this rather than being too clever in the JS.

**Comment by ianmcburnie** [comment]
This got fixed in 0.1.0
